src/frameworks/index.ts

```diff
@@ -538,6 +539,12 @@ ${
 
       // TODO move to templates
 
+      if (frameworksBackend && framework === "next") {
+        if (await isUsingImageOptimization(getProjectPath(), ".next")) {
```

Comment:
`.next` can be customized, so we should use `distDir` instead, which should be easy after we move this to `ɵcodegenFunctionsDirectory` as we have it there

Reply:
Not needed anymore since the check is happening in the Next.js file.